scripts/unit-tests.sh

```diff
@@ -6,9 +6,8 @@ set -a
 source .env
 set +a 
 
-# Launch anvil in the bg
 yarn anvil:kill
-yarn anvil > anvil-unit.log 2>&1 &
+yarn anvil --chain-id 1337 > anvil-unit.log 2>&1 &
```

Comment:
failing if not using hardhat chainid bc mocking intercepts all of swaps now

Reply:
we can make the ones not mocked go thru if needed

Reply:
is there a benefit to that? since all are passing now

Reply:
nop